server/fsm.go

```diff
@@ -696,13 +696,25 @@ func (h *FSMHandler) recvMessageWithError() (*FsmMsg, error) {
 				}
 			case bgp.BGP_MSG_NOTIFICATION:
 				body := m.Body.(*bgp.BGPNotification)
-				log.WithFields(log.Fields{
-					"Topic":   "Peer",
-					"Key":     h.fsm.pConf.Config.NeighborAddress,
-					"Code":    body.ErrorCode,
-					"Subcode": body.ErrorSubcode,
-					"Data":    body.Data,
-				}).Warn("received notification")
+				if body.ErrorCode == bgp.BGP_ERROR_CEASE && body.ErrorSubcode == bgp.BGP_ERROR_SUB_ADMINISTRATIVE_SHUTDOWN {
+					log.WithFields(log.Fields{
+						"Topic":         "Peer",
+						"Key":           h.fsm.pConf.Config.NeighborAddress,
+						"Code":          body.ErrorCode,
+						"Subcode":       body.ErrorSubcode,
+						"Lenght":        body.Data[0],
```

Comment:
Thanks!
Yes, It is a typo...

Also, Travis-CI says my patch collapses some other features,
and I'm investigating why.